### ISSUE TITLE
[TabContainer]: [Browser Viewport Bug]

### ISSUE THREAD

**Issue by florianschmidtsap** [SAP/ui5-webcomponents-react#5455]
### Describe the bug

I am using an ObjectPage an inside that a TabContainer. Everytime I switch between Tabs, the browser changes its view (feels like it is automatically scrolling up). In that way the Tab itself is not on screen anymore. The whole app is rerendered (I guess?) and then I start at the upper Section of the Object Page.

### Reproduction steps

1. Create ObjectPage
2. Add SubSections
3. Lowest SubSection contains a TabContainer which is outside the view on first render
4. Switch between Tabs
5. See chaging View

### Expected Behaviour

Stay in the same view just change the Tab

### UI5 Web Components for React Version

1.9.3

### UI5 Web Components Version

1.9.3

### Browser

Chrome

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Hi @florianschmidtsap 

please add a reproducible example, preferably by using this [codeSandbox ](https://codesandbox.io/p/devbox/determined-ganguly-rcpmd4) template.

**Comment by florianschmidtsap** [comment]
Here you go. Thanks for your support

https://codesandbox.io/p/devbox/exciting-solomon-lypfcl?file=%2Fsrc%2FApp.jsx%3A54%2C103&workspaceId=138294c4-0b36-4882-a6d2-d62da9c60cc8

**Comment by Lukas742** [comment]
Apparently drafts are no longer shareable at codeSandbox since January 30th, so I don't have access to it...

> From Jan 30th, all draft Sandboxes need to be private. To edit this Sandbox, you must convert it into a personal draft by making it private (it will become non-shareable and non-embeddable).

Unfortunately we only realized this now.

Can you please fork this [StackBlitz sandbox](https://stackblitz.com/github/SAP/ui5-webcomponents-react/tree/main/examples/vite-ts?file=src%2FApp.tsx) and implement the example there again? Sorry for the inconvenience.

**Comment by florianschmidtsap** [comment]
`      <ObjectPage>
        <ObjectPageSection>
          <Label>Hello World</Label>
          <p>
            Lorem ipsum dolor sit amet consectetur adipisicing elit. Accusamus
            maiores quisquam vitae qui pariatur illo eius porro dignissimos
            ipsum, adipisci repellat. Explicabo corporis voluptas ut veritatis.
            Blanditiis eius perspiciatis illo!
          </p>
        </ObjectPageSection>
        <ObjectPageSection>
          <Label>Hello World</Label>
          <p>
            {" "}
            Lorem ipsum dolor sit amet consectetur adipisicing elit. Iure
            placeat molestiae incidunt quisquam quia voluptatum commodi
            quibusdam officia impedit quo soluta neque, eveniet enim praesentium
            similique voluptas. Cumque, quas ullam.
          </p>
        </ObjectPageSection>
        <ObjectPageSection>
          <Label>Hello World</Label>
        </ObjectPageSection>
        <ObjectPageSection>
          <Label>Hello World</Label>
          <TabContainer>
            <Tab text="Test">Test</Tab>
            <Tab text="Test">Test</Tab>
            <Tab text="Test">Test</Tab>
            {/* on changing this tabs, the viewport of the browser gets resetted / scrolled to the top */}
          </TabContainer>
        </ObjectPageSection>
      </ObjectPage>`
      
      Basically thats the Code, to see the result, just make sure that you have to scroll down to see the Tabs. Everytime you change the tab you get back to the top of the page.

**Comment by Lukas742** [comment]
> on changing these tabs, the browser jumps back to the top, not here what is pretty weird but in my app which has the same accept code

This is probably because you're using an outdated version (v1.9.3). Please update your dependencies and check again.
